docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Panel
+description: A common, versatile pattern used to display content composed of different elements, usually in a collection.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Panel/Panel.tsx
+---
+
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  Button,
+  Icon
+} from '@deque/cauldron-react';
+
+```js
+import { Panel, PanelHeader, PanelContent } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Panel Component
+
+At its simplest, a `Panel` is a styled semantic HTML `<section>` element with with content inside.
+
+<Note>
+  If your section needs to have an accessible name, be sure to include an
+  `aria-label` or `aria-labelledby` pointed to the `id` of your label.
+</Note>
+
+```jsx example
+<Panel aria-label="Panel Heading">Panel Content</Panel>
```

Comment:
We should make this something unique since _Panel Heading_ is used below, maybe _Panel Content_? This would get rid of a duplicate accessible name.